pkg/transport/transport.go

```diff
@@ -100,7 +104,7 @@ func (transport *Transport) DialAddress(ctx context.Context, address string, opt
 	defer mon.Task()(&ctx)(&err)
 
 	options := append([]grpc.DialOption{
-		transport.tlsOpts.DialOption(storj.NodeID{}),
+		transport.tlsOpts.DialUnverifiedIDOption(),
```

Comment:
I am not an expert in gRPC, but can we just skip this dial option instead of having the somewhat artifical `DialUnverifiedIDOption`?

Reply:
We still need verify peercerts and similar things.

Reply:
https://github.com/storj/storj/pull/1395/files/3099ebef2ed134ac24f25ad3ef8fa6db48baa0d5#diff-26668019454cf4ea9a94466a211afb71R72

Reply:
OK